src/main/java/uk/gov/pay/api/model/PaymentState.java

```diff
@@ -24,28 +24,33 @@
 
     @JsonProperty("code")
     private String code;
+    
+    @JsonProperty("details")
+    private String details;
 
     public static PaymentState createPaymentState(JsonNode node) {
         return new PaymentState(
                 node.get("status").asText(),
                 node.get("finished").asBoolean(),
                 node.has("message") ? node.get("message").asText() : null,
-                node.has("code") ? node.get("code").asText() : null
+                node.has("code") ? node.get("code").asText() : null,
+                node.has("details") ? node.get("details").asText() : null
         );
     }
 
     public PaymentState() {
     }
 
     public PaymentState(String status, boolean finished) {
-        this(status, finished, null, null);
+        this(status, finished, null, null, null);
```

Comment:
I think maybe it's worthwhile having a subclass for direct debit payments states that add the "details" field to make it clear that this is just for direct debits